docs/03-extensions/magento1/how-to/setup-magento-extension.mdx

```diff
@@ -0,0 +1,295 @@
+---
+sidebar_position: 1
+title: Set-up Magento Extension
```

Comment:
This can be missleading when using "Extension", when first seeing this in the docs, I thought it made reference to a [application extension](https://developers.front-commerce.com/docs/remixed/guides/register-an-extension).

```suggestion
title: Set-up Front-Commerce integration in Magento
```

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/49dd4f74-6887-4802-9404-9ba7e49f0286)

Reply:
Integration is a specific term for M2, since in the setup doc we refer to module, i would go for module instead WDYT ?

Reply:
Replaced with module